src/coreclr/jit/flowgraph.cpp

```diff
@@ -3557,6 +3573,11 @@ PhaseStatus Compiler::fgDetermineFirstColdBlock()
     for (block = firstColdBlock; block != nullptr; block = block->bbNext)
     {
         block->bbFlags |= BBF_COLD;
+
+        if (forceSplit)
```

Comment:
We should `unmarkLoopAlign` regardless of `forceSplit`. In other words, we should always disable loop alignment for cold blocks.

Reply:
Fixed, thanks for catching that!